cypress/integration/other/configuration.spec.js

```diff
@@ -126,4 +126,26 @@ describe('Configuration', () => {
       );
     });
   });
+
+  describe('suppressErrorRendering', () => {
+    it('should not render error diagram if suppressErrorRendering is set', () => {
+      const url = 'http://localhost:9000/suppressError.html?suppressErrorRendering=true';
+      cy.viewport(1440, 1024);
+      cy.visit(url);
+      cy.get('#test');
```

Comment:
I'm getting a race-condition when running the E2E test. The following seems to fix it:

```suggestion
      cy.get('#test').find("svg").should(($svg) => {
        expect($svg).to.have.length(2); // all failing diagrams should not appear!
        $svg.each((_index, svg) => {
          expect(cy.$$(svg)).to.be.visible;

          // none of the diagrams should be error diagrams
          expect($svg).to.not.contain("Syntax error");
        });
      });
```

### See screenshot of before and after:

![configuration spec-should-not-render-error-diagram-if-suppressErrorRendering-is-set diff](https://github.com/mermaid-js/mermaid/assets/19716675/61d57753-6e8e-4f78-b056-7d0aa30f2690)